addons/mail/static/src/scss/mail_activity.scss

```diff
@@ -141,8 +141,11 @@
             }
         }
 
-        .o_activity_link_kanban {
-            visibility: hidden;
+        // hiding only in larger screen devices
+        @media (min-width: $screen-md-max) {
```

Comment:
Should be `$screen-xs-max` for mobile.

Most css styles use XS for mobile. Recent commit sets new mobile size to SM (https://github.com/odoo/odoo/commit/4e5f826ce53b9e4cb73eea77935ea82e81ac66b9), but I think this is a mistake.